Controller/ChangePasswordFOSUser1Controller.php

```diff
@@ -33,7 +33,7 @@ public function changePasswordAction()
     {
         $user = $this->getUser();
         if (!is_object($user) || !$user instanceof UserInterface) {
-            $this->createAccessDeniedException('This user does not have access to this section.');
+            throw $this->createAccessDeniedException('This user does not have access to this section.');
```

Comment:
Then I would just use `throw new AccessDeniedException('This user does not have access to this section.')` on all of them